contracts/estate/EstateRegistry.sol

```diff
@@ -192,6 +197,28 @@ contract EstateRegistry is Migratable, IEstateRegistry, ERC721Token, ERC721Recei
     emit UpdateOperator(estateId, operator);
   }
 
+  /**
+   * @notice Set estates updateOperator
+   * @param _estateIds - Estate ids
+   * @param _operator - address of the account to be set as the updateOperator
+   */
+  function setManyUpdateOperator(
+    uint256[] _estateIds,
+    address _operator
+  )
+    public
+  {
+    for (uint i = 0; i < _estateIds.length; i++) {
+      setUpdateOperator(_estateIds[i], _operator);
+    }
+  }
+
+  /**
+   * @notice Set many lands updateOperator
```

Comment:
There is a typo here, should be `Set LAND updateOperator`

Reply:
```suggestion
   * @notice Set LAND updateOperator
```